tests/baselines/reference/restTupleElements1.errors.txt

```diff
@@ -94,14 +92,12 @@ tests/cases/conformance/types/tuple/restTupleElements1.ts(59,4): error TS2345: A
                                   ~~~~~~~~~~~~~~~~~~~~~~~~
 !!! error TS2344: Type '[number, number, string]' does not satisfy the constraint '[number, ...number[]]'.
 !!! error TS2344:   Type at positions 1 through 2 in source is not compatible with type at position 1 in target.
-!!! error TS2344:     Type 'string | number' is not assignable to type 'number'.
-!!! error TS2344:       Type 'string' is not assignable to type 'number'.
+!!! error TS2344:     Type 'string' is not assignable to type 'number'.
     assign<[number, ...number[]], [number, number, number, string]>();  // Error
                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 !!! error TS2344: Type '[number, number, number, string]' does not satisfy the constraint '[number, ...number[]]'.
 !!! error TS2344:   Type at positions 1 through 3 in source is not compatible with type at position 1 in target.
-!!! error TS2344:     Type 'string | number' is not assignable to type 'number'.
-!!! error TS2344:       Type 'string' is not assignable to type 'number'.
+!!! error TS2344:     Type 'string' is not assignable to type 'number'.

```

Comment:
Same here, given what we've had before, I think this is good. It actually might be more clear than the previous message, because before we had `type 'string | number'` mentioned, but the original code doesn't really have that union type, so that could be confusing.

Reply:
Yeah, I also found the union type being reported here kinda unintuitive.